apps/omg_watcher/lib/omg_watcher/api/account.ex

```diff
@@ -36,4 +36,12 @@ defmodule OMG.Watcher.API.Account do
   def get_utxos(address) do
     DB.TxOutput.get_utxos(address)
   end
+
+  @spec get_exitable_utxos(OMG.Crypto.address_t()) :: list(%DB.TxOutput{})
+  def get_exitable_utxos(address) do
```

Comment:
document using `@doc` mentioning the intent and properities and differences to `get_utxos`.

Also document the new endpoint in the appropriate API documents

Reply:
> Also document the new endpoint in the appropriate API documents

:point_up: what about this? I only see a rename in the API documentation

Reply:
https://github.com/omisego/elixir-omg/blob/f45dd822c6949ace5d137ad735400afc78ddd36f/apps/omg_watcher/priv/swagger/watcher_api_specs.yaml#L49
it is already documented

Reply:
ah, I see now. But anyway I think its `account.get_exitable_utxos`